examples/webgl_materials_envmaps_exr.html

```diff
@@ -79,14 +78,15 @@
 					.load( 'textures/piz_compressed.exr', function ( texture ) {
 
 						texture.minFilter = THREE.NearestFilter;
-						// texture.magFilter = THREE.NearestFilter;
 						texture.encoding = THREE.LinearEncoding;
 
-						var cubemapGenerator = new EquirectangularToCubeGenerator( texture, { resolution: 512, type: THREE.HalfFloatType } );
-						exrBackground = cubemapGenerator.renderTarget;
-						var cubeMapTexture = cubemapGenerator.update( renderer );
+						var options = {
+							minFilter: THREE.NearestFilter
```

Comment:
I'm not sure that makes sense. Here, or in the lines above. Probably should be `Linear`.

You will have to step through the code to see if it matters.

Also, you will almost surely have to change the loader output data type to `HalfFloat` to duplicate the previous results.
```js
.setDataType( THREE.HalfFloatType )
```

Reply:
> I'm not sure that makes sense. Here, or in the lines above. Probably should be Linear.

Why do the previous texture filtering settings are wrong. It does actually look good.

> Also, you will almost surely have to change the loader output data type to HalfFloat to duplicate the previous results.

That's true. Let me fix this!

Reply:
Links for direct comparison:

https://threejs.org/examples/webgl_materials_envmaps_exr
https://raw.githack.com/Mugen87/three.js/dev39/examples/webgl_materials_envmaps_exr.html

Reply:
Your example is throwing errors on my machine.

I have always used Linear interpolation when the encoding supports it. Some encodings like RGBE are not compatible with linear, so nearest must be used. I have never seen Linear and Nearest mixed in the same texture. If there is a use case for that, I am not aware of it.

Reply:
> Your example is throwing errors on my machine.

Um, can't reproduce this on my iMac with Chrome. What errors are logged?

Reply:
Failure is on macOS Safari.
>WebGL: INVALID_OPERATION: texImage2D: type HALF_FLOAT_OES but ArrayBufferView is not NULL

Reply:
Okay, I can reproduce. It seems `WebGLRenderTargetCube.fromEquirectangularTexture()` throws this error on Safari when the input texture is of `THREE.HalfFloatType`. Using `THREE.FloatType` with `EXRLoader` as before solves the issue. However, since `WebGLRenderTargetCube.fromEquirectangularTexture()` strictly derives the `type` of the source texture (unlike `EquirectangularToCubeGenerator` which does `type: options.type || this.sourceTexture.type`), I can't fix this issue without changing previous values (see https://github.com/mrdoob/three.js/pull/17708#issuecomment-541005429).

Reply:
see #17727